tests/check/check_test.go

```diff
@@ -54,25 +49,30 @@ func testCheck(t *testing.T, engine string) {
 	require.NoError(t, err)
 	defer conn.Close()
 
-	RunSchema1_1CheckTests(t, pb.NewOpenFGAServiceClient(conn))
-	RunSchema1_0CheckTests(t, pb.NewOpenFGAServiceClient(conn))
+	t.Run("RunAll", func(t *testing.T) {
+		t.Run("testCheck", func(t *testing.T) {
```

Comment:
I'd omit the `test` prefix. It just stutters. Lets name as `Check` and `BadAuthModelID`. So we'll get, for example:

`TestCheckMemory/RunAll/Check`
`TestCheckMemory/RunAll/BadAuthModelID`